blocks/v2-embed/v2-embed.js

```diff
@@ -23,29 +23,31 @@ const extractAspectRatio = (block) => {
     return null;
   }
 
-  const match = aspectRatioClass.match(aspectRatioRegex);
-  if (match) {
-    return {
-      width: parseInt(match[1], 10),
-      height: parseInt(match[2], 10),
-    };
-  }
+  const match = aspectRatioClass?.match(aspectRatioRegex);
 
-  return null;
+  return match
+    ? {
+        width: parseInt(match[1], 10),
+        height: parseInt(match[2], 10),
+      }
+    : null;
 };
 
-const retrieveVideoConfig = (block, aspectRatio) => ({
-  ...(aspectRatio ? { aspectRatio: `${aspectRatio.width}:${aspectRatio.height}` } : {}),
-  ...(block.querySelector('img')?.getAttribute('src')
-    ? { poster: new URL(block.querySelector('img').getAttribute('src'), window.location.href).href }
-    : {}),
-  autoplay: block.classList.contains('autoplay') ? 'any' : false,
-  muted: block.classList.contains('autoplay'),
-  loop: block.classList.contains('loop'),
-  controls: !block.classList.contains('disable-controls'),
-  disablePictureInPicture: block.classList.contains('disable-picture-in-picture'),
-  language: document.documentElement.lang,
-});
+const retrieveVideoConfig = (block, aspectRatio) => {
```

Comment:
if this function is in video-helper, is it worth it to export it and use it here?

Reply:
after checking them both, are different enough to have it better like it is